webui/react/src/components/kit/LogViewer/LogViewer.test.tsx

```diff
@@ -282,16 +282,18 @@ describe('LogViewer', () => {
     });
 
     it('should render logs with streaming', async () => {
-      setup({ decoder, onFetch });
-
-      await waitFor(
-        () => {
-          const lastLog = logsReference[logsReference.length - 1];
-          expect(lastLog.message).not.toBeNull();
-          expect(screen.queryByText(lastLog.message)).toBeInTheDocument();
-        },
-        { timeout: 6000 },
-      );
+      if (process.env.QUARANTINED) {
```

Comment:
**question:** do we want to skip the test when `QUARANTINED` is true?

Reply:
✅  added using `global.quarantinedIt`

Reply:
ok I had to rework how it's imported